server/sonar-db-dao/src/main/java/org/sonar/db/permission/AuthorizationDao.java

```diff
@@ -131,7 +131,7 @@ public int countUsersWithGlobalPermissionExcludingUserPermission(DbSession dbSes
         if (userId == null) {
           return mapper(dbSession).keepAuthorizedProjectIdsForAnonymous(role, componentIds);
         }
-        return mapper(dbSession).keepAuthorizedProjectIdsForUser(userId, role, componentIds);
+        return mapper(dbSession).keepAuthorizedProjectIdsForUser(userId, role, componentIds);//FIXME chunk size
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Take the required action to fix the issue indicated by this comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1134)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ATrailingCommentCheck)